src/app/modules/core/services/study.service.ts

```diff
@@ -26,9 +26,27 @@ export class StudyService  {
   study$ = this.SelectedReturn.asObservable()
   //SET MASS
     public setConcentration(mass) {
-      this.selectedStudy.SpillMass = mass;
-      this.SelectedReturn.next(this.selectedStudy);
+        this.selectedStudy.SpillMass = mass;
+        this.SelectedReturn.next(this.selectedStudy);
     }
+  //Return output from study service
+  public roundOutputs() { //ROUND  output numbers;
+    var reaches: Array<any> = [];
+    reaches = Object.values(this.selectedStudy.Results["reaches"]);
+    reaches.shift();
+    reaches.forEach((o => {
```

Comment:
All concentration values are rounded in the services. It looks like this is to round our final values but it shouldn't be needed.